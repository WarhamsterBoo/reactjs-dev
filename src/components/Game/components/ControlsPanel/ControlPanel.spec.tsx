import React from "react";
import renderer from "react-test-renderer";
import { ControlPanel } from ".";

describe("ControlPanel", () => {
  it("should render", () => {
    const sut = <ControlPanel />;

    expect(renderer.create(sut).toJSON()).toMatchSnapshot();
  });
});
