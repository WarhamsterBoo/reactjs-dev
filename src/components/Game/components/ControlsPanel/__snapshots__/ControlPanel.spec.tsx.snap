// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlPanel should render 1`] = `
.emotion-5 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  border: solid 1px black;
  padding: 10px 10px 10px 10px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-0 {
  margin-left: 7px;
  margin-right: 7px;
  border-radius: 4px;
}

<div
  className="emotion-5"
>
  <button
    className="emotion-0"
  >
    Stop
  </button>
  <button
    className="emotion-0"
  >
    Pause
  </button>
  <button
    className="emotion-0"
  >
    Faster
  </button>
  <button
    className="emotion-0"
  >
    1x
  </button>
  <button
    className="emotion-0"
  >
    Slower
  </button>
</div>
`;
